dev/generate_tokens/base_tokens_map.js

```diff
@@ -23,174 +23,174 @@ module.exports = {
-      ChainId.AURORA,
-      '0xf4eb217ba2454613b15dbdea6e5f22276410e89e',
-      8,
-      'WBTC',
-      'Wrapped Bitcoin'
-    )
-  },
-
-  DAI: {
-    [ChainId.AVALANCHE]: new Token(
-      ChainId.AVALANCHE,
-      '0xbA7dEebBFC5fA1100Fb055a87773e1E99Cd3507a',
-      18,
-      'DAI',
-      'Dai Stablecoin'
-    ),
-    [ChainId.POLYGON]: new Token(
-      ChainId.POLYGON,
-      '0x8f3cf7ad23cd3cadbd9735aff958023239c6a063',
-      18,
-      'DAI',
-      'Dai Stablecoin'
-    ),
-    [ChainId.AURORA]: new Token(
-      ChainId.AURORA,
-      '0xe3520349f477a5f6eb06107066048508498a291b',
-      18,
-      'DAI',
-      'Dai Stablecoin'
-    )
-  },
-
-  TRI: {
-    [ChainId.AURORA]: new Token(
-      ChainId.AURORA,
-      '0xFa94348467f64D5A457F75F8bc40495D33c65aBB',
-      18,
-      'TRI',
-      'Trisolaris Token'
-    )
-  },
-
-  AURORA: {
-    [ChainId.AURORA]: new Token(ChainId.AURORA, '0x8BEc47865aDe3B172A928df8f990Bc7f2A3b9f79', 18, 'AURORA', 'Aurora')
-  },
-
-  UST: {
-    [ChainId.AURORA]: new Token(
-      ChainId.AURORA,
-      '0x5ce9F0B6AFb36135b5ddBF11705cEB65E634A9dC',
-      18,
-      'atUST',
-      'Wrapped UST'
-    )
-  },
-
-  LUNA: {
-    [ChainId.AURORA]: new Token(
-      ChainId.AURORA,
-      '0xC4bdd27c33ec7daa6fcfd8532ddB524Bf4038096',
-      18,
-      'atLUNA',
-      'Wrapped LUNA'
-    )
-  },
-
-  USDC: {
-    [ChainId.AVALANCHE]: new Token(
-      ChainId.AVALANCHE,
-      '0xA7D7079b0FEaD91F3e65f86E8915Cb59c1a4C664',
-      6,
-      'USDC',
-      'USD Coin'
-    ),
-    [ChainId.POLYGON]: new Token(ChainId.POLYGON, '0x2791bca1f2de4661ed88a30c99a7a9449aa84174', 6, 'USDC', 'USD Coin')
-  },
-
-  AAVE: {
-    [ChainId.AVALANCHE]: new Token(
-      ChainId.AVALANCHE,
-      '0x63a72806098Bd3D9520cC43356dD78afe5D386D9',
-      18,
-      'AAVE',
-      'AAVE Token'
-    ),
-    [ChainId.POLYGON]: new Token(
-      ChainId.POLYGON,
-      '0xd6df932a45c0f255f85145f286ea0b292b21c90b',
-      18,
-      'AAVE',
-      'AAVE Token'
-    )
-  },
-
-  WNEAR: {
-    [ChainId.AURORA]: new Token(
-      ChainId.AURORA,
-      '0xc42c30ac6cc15fac9bd938618bcaa1a1fae8501d',
-      24,
-      'wNEAR',
-      'WNEAR Token'
-    )
-  },
-
-  MECHA: {
-    [ChainId.AURORA]: new Token(ChainId.AURORA, '0xa33C3B53694419824722C10D99ad7cB16Ea62754', 18, 'MECHA', 'Mecha')
-  },
-
-  STNEAR: {
-    [ChainId.AURORA]: new Token(
-      ChainId.AURORA,
-      '0x07f9f7f963c5cd2bbffd30ccfb964be114332e30',
-      24,
-      'stNEAR',
-      'Staked NEAR'
-    )
-  },
-
-  META: {
-    [ChainId.AURORA]: new Token(ChainId.AURORA, '0xc21ff01229e982d7c8b8691163b0a3cb8f357453', 24, 'META', 'Meta Pool')
-  },
-
-  GBA: {
-    [ChainId.AURORA]: new Token(
-      ChainId.AURORA,
-      '0xc2ac78FFdDf39e5cD6D83bbD70c1D67517C467eF',
-      18,
-      'GBA',
-      'Golden Banana'
-    )
-  },
-
-  XNL: {
-    [ChainId.AURORA]: new Token(ChainId.AURORA, '0x7cA1C28663b76CFDe424A9494555B94846205585', 18, 'XNL', 'Chronicle')
-  }
+  // USDT: {
```

Comment:
Do we not need these anymore? any reason its commented out

Reply:
base_tokens_map is a file for overriding the tokens that are generated when running ``yarn build tokens``, which fetches the tokens repo and builds the token constants.

 As I understand it, there is no need to override anything in the tokens repo here now

Reply:
Please don't commit commented out code to production.

Reply:
When building the token generator, we needed to ensure the token metadata was stable between changes.  To do this, I created `https://github.com/trisolaris-labs/interface/blob/main/src/state/stake/__tests__/stake-constants.test.ts` and `https://github.com/trisolaris-labs/interface/blob/main/src/constants/tokens/__tests__/index.test.ts`

The PR can be found here: https://github.com/trisolaris-labs/interface/pull/152

Please confirm that your new data is correct, and if it is, update the snapshot tests by running `yarn test -u`

Reply:
Removed🙏